content/03.methods.md

```diff
@@ -114,7 +119,7 @@ We reported the half-life of each bioRxiv preprint category.
 We hypothesized that preprints would be more likely to be published in journals that contained similar content to the work in question.
 To test this hypothesis, we designed an experiment examining document and journal representations.
 First, we removed all journals that had fewer than 100 papers in the PMC corpus.
-A subset of our PMC corpus was directly linked to papers in bioRxiv as they had been published as open access articles.
+A subset of our PMC corpus was directly linked to papers in bioRxiv as they had been published as open access articles (see Mapping bioRxiv preprints to their published counterparts).

```

Comment:
```suggestion
Certain manuscripts in the PMC corpus were annotated to their corresponding bioRxiv preprints through CrossRef as previously noted.
```

Reply:
These felt pretty close to the CrossRef mention, so I less formally cross-referenced them :)